source/AI.cpp

```diff
@@ -3321,7 +3321,7 @@ void AI::MovePlayer(Ship &ship, const PlayerInfo &player, Command &activeCommand
 			else if(object.HasSprite())
 			{
 				double distance = ship.Position().Distance(object.Position());
-				if(distance < object.Radius())
+				if(distance < object.Radius() && ship.Velocity().Length() < (80. / 60.))
```

Comment:
Any specific reason behind choosing a velocity of 80 (as seen in game)? Why not some factor of a ship's max velocity? If we stick with the single value that is considered low speed for all ships then it's probably worth defining this as a variable so that it's a bit more clear what exactly the value represents.

Reply:
>It is currently based on having a speed less than 80 pixels/second, which is just below the lowest top speeds of stock ships (such as the Skein and Bulk Freighter).

Went with this reasoning for it, with it being basically stationary. I'll throw it in a variable, though.